naviswarm/Scan scan_now
